runtime/src/main/java/org/corfudb/runtime/view/TableRegistry.java

```diff
@@ -293,6 +302,16 @@ public static String getFullyQualifiedTableName(TableName tableName) {
         return getFullyQualifiedTableName(tableName.getNamespace(), tableName.getTableName());
     }
 
+    /**
+     * Return the stream Id for the provided stream tag.
+     *
+     * @param streamTag stream tag in string
+     * @return stream Id in UUID
+     */
+    public static UUID getStreamIdForStreamTag(String streamTag) {
+        return CorfuRuntime.getStreamID(STREAM_TAG_PREFIX + streamTag);
+    }
```

Comment:
Shouldn't this be scoped to a namespace (i.e., STREAM_TAG_PREFIX + nameSpace + streamTag) ?

There could be an issue were subscribers can read data from a different namespace.

Reply:
Done, initially I expect streamTags to be unique, but yes having a namespace is better.